test/t_web/channels/feed_channel_test.exs

```diff
@@ -72,17 +72,18 @@ defmodule TWeb.FeedChannelTest do
                  "id" => m3.id,
                  "profile" => %{name: "mate-3", story: [], user_id: p3.id, gender: "M"},
                  "timeslot" => %{"selected_slot" => s2},
-                 "audio_only" => false
+                 "audio_only" => false,
+                 "expiration_date" => ~U[2021-10-07 12:16:07Z]
                },
                %{
                  "id" => m2.id,
                  "profile" => %{name: "mate-2", story: [], user_id: p2.id, gender: "N"},
-                 "contact" => %{
-                   "contacts" => %{"whatsapp" => "+79666666666"},
-                   "picker" => p2.id,
-                   "opened_contact_type" => nil
+                 "timeslot" => %{
+                   "picker" => me.id,
+                   "slots" => slots
```

Comment:
@akhrail1996 it appears that this test was wrong, according to [the comment above](https://github.com/getsince/test3/blob/c938e8ae9200cac79c518822e310e92411379ab7/test/t_web/channels/feed_channel_test.exs#L60)

Reply:
@ruslandoga actually it's the other way around - test is correct, but comment was wrong 🙃